tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -144,17 +149,19 @@ private static ArgumentParser argParser() {
 
         parser.addArgument("--num-threads")
                 .action(store())
-                .required(true)
+                .required(false)
                 .type(Integer.class)
                 .metavar("NUM-THREADS")
                 .dest("numThreads")
-                .help("The value bound of the random integers in message payload to simulate different compression ratio.");
+                .setDefault(1)
+                .help("The number of producing threads.");
 
         parser.addArgument("--value-bound")
                 .action(store())
-                .required(true)
+                .required(false)
                 .type(Integer.class)
                 .metavar("VALUE-BOUND")
+                .setDefault(0)
                 .dest("valueBound")
                 .help("The value bound of the random integers in message payload to simulate different compression ratio.");
```

Comment:
I am thinking that, without any idea of how the ratio is affected by this config, users will choose either 0 or MAX_INT. Not a big deal for me if you don't bother to get it.